demos/storybook/stories/dropdown-toolbar/with-custom-menu.tsx

```diff
@@ -0,0 +1,63 @@
+import { DropdownToolbar, InfoListItem } from '@pxblue/react-components';
+import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types';
+import React from 'react';
+import { AppBar, Menu } from '@material-ui/core';
+import { Business, House, Apartment } from '@material-ui/icons';
+import * as Colors from '@pxblue/colors';
+import { action } from '@storybook/addon-actions';
+import { Store, State } from '@sambego/storybook-state';
+
+type DropdownMenuState = {
+    open: boolean;
+};
+const store = new Store<DropdownMenuState>({
+    open: false,
+});
+
+export const withCustomMenu = (): StoryFnReactReturnType => {
```

Comment:
Looks good...can we throw a style somewhere (`outline: none`) to get rid of this active border state? 
![image](https://user-images.githubusercontent.com/29152776/90506385-2c80d600-e122-11ea-824d-2b0714e90549.png)

Maybe just on the InfoListItem component itself?